package.json

```diff
@@ -29,15 +29,15 @@
     "url": "https://github.com/Pomax/react-onclickoutside/issues"
   },
   "scripts": {
-    "build": "npm run build:es && npm run build:commonjs",
+    "build": "run-p build:*",
     "build:es": "cross-env BABEL_ENV=es babel src -d es",
     "build:commonjs": "cross-env BABEL_ENV=commonjs babel src -d lib",
     "lint": "eslint src/*.js ./test",
     "test": "run-s test:**",
-    "test:basic": "run-s lint build",
+    "test:basic": "run-s lint build:commonjs",
```

Comment:
yeah, probably we should, cannot spare time now to configure it though :/ also it only would test other module format, no other code changes, `es` build is all es5 except of `import`/`export`syntax